definitions/npm/jest_v14.0.x/flow_>=v0.23.x/jest_v14.0.x.js

```diff
@@ -0,0 +1,71 @@
+// flow-typed signature: e2130120dcdc34bf09ff82449b0d508c
+// flow-typed version: 230d7577ce/jest_v12.0.x/flow_>=v0.23.x
+
+type JestMockFn = {
+  (...args: Array<any>): any;
+  mock: {
+    calls: Array<Array<any>>;
+    instances: any;
```

Comment:
>  Can we merge this and follow-up later?

We aim to keep a high quality bar for merging libdefs because we want that to be an expectation of the users of this repository.

Merging types like `any` in places like this are especially prone to degrading that expectation since it's pretty easy to never notice as they basically eliminate all possibilities of giving an error.